reporting/miro_transformer/src/transform.py

```diff
@@ -0,0 +1,83 @@
+import json
+from datetime import datetime
+
+
+def transform(record):
+    """
+    Parameters
+    ----------
+    record : dict
+        the raw data from VHS in a malleable dict format
+
+    Returns
+    -------
+    transformed_record : dict
+        record with necessary transformations applied, ready to be formatted for
+        elasticsearch ingestion
+    """
+    original_data = json.loads(record["data"])
+    transformed = drop_redundant_fields(original_data, keys_to_drop)
+    transformed = clean_dates(transformed)
+    return transformed
+
+
+def drop_redundant_fields(original_data, keys_to_drop):
+    """
+    strip out any of the original VHS fields which are either empty, sparse,
+    redundantly filled, or just generally unwanted in our ES index
+    """
+    clean_data = {
+        key: value for key, value in original_data.items() if key not in keys_to_drop
+    }
+    return clean_data
+
+
+def clean_dates(data):
+    """
+    elasticsearch's default sort is alphabetical, and sorting dd/mm/yyyy dates
+    alphabetically is rubbish
+    """
+    for key, value in data.items():
+        if "date" in key and value is not None:
+            if type(value) == str:
```

Comment:
minor: when comparing types in Python, it's better to use `isinstance` rather than `type()`, because then you cover inheritance as well.
```suggestion
            if isinstance(value, str):
```